tethys_apps/harvester.py

```diff
@@ -80,14 +80,24 @@ def get_url_patterns(self):
         """
         app_url_patterns = dict()
         extension_url_patterns = dict()
+        routing_url_patterns = dict()
 
         for app in self.apps:
-            app_url_patterns.update(app.url_patterns)
+            app_url_patterns.update(app.url_patterns['http'])
+
+        for app in self.apps:
+            routing_url_patterns.update(app.url_patterns['websocket'])
 
         for extension in self.extensions:
-            extension_url_patterns.update(extension.url_patterns)
+            extension_url_patterns.update(extension.url_patterns['http'])
+
+        url_patterns = {
+            'url_pattern': app_url_patterns,
```

Comment:
Let's have a common naming scheme for these keys:
app_url_patterns
ext_url_patterns
ws_url_patterns

Reply:
Change the keys of the url_patterns dictionary to these names.